suricata.yaml.in

```diff
@@ -1091,6 +1091,18 @@ asn1-max-frames: 256
 #  user: suri
 #  group: suri
 
+# Use landlock security module under Linux
+landlock:
+  enabled: no
+  directories:
+    write:
+      - @e_logdir@
```

Comment:
Couldn't the log and run directory be a static/define string?

Reply:
the are set by `--sysconfdir` and `--localstatedir`, e.g. `./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var`

Reply:
These values it can be overwritten from configuration file so I prefer to have them here. For the log directory @jasonish remark on -l usage makes sense and it could be better to avoid to add it in the configuration.

Reply:
I got in down to 
```yaml
  directories:
    #write:
    #  - @e_rundir@
    read:
      - /usr/
      - /etc/
      - @e_sysconfdir@
```

Reply:
> the are set by `--sysconfdir` and `--localstatedir`, e.g. `./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var`

By static/define I meant C `static` or `#define` strings that could be populated by the `./configure` command at build time.

Reply:
> I got in down to in the MR to come
> 
> ```yaml
>   directories:
>     #write:
>     #  - @e_rundir@
>     read:
>       - /usr/
>       - /etc/
>       - @e_sysconfdir@
> ```

Great to not have to worry about write permissions!

`/usr` and `/etc` could maybe be inferred from a build configuration, maybe a new one with these paths as default values? I think it would be OK if users only had to specify such paths when they have a non-standard installation or distro.